CHANGELOG.md

```diff
@@ -4,6 +4,7 @@ FEATURES:
 
 * `vcd_vapp_vm` - Ability to add metadata to a VM. For previous behaviour please see `BACKWARDS INCOMPATIBILITIES` [GH-158]
 * `vcd_vapp_vm` - Ability to enable hardware assisted CPU virtualization for VM. It allows hypervisor nesting. [GH-219]
+* **New Resource:** VDC resource `vcd_external_network` - [GH-230]
```

Comment:
```suggestion
* **New Resource:** VCD resource `vcd_external_network` - [GH-230]
```

Reply:
Overall, we should always use capitalisation "vCD" not "VCD". However, we don't need to emphasise that as everything is for "vCD" here.

I would suggest changing to word "provider" instead as to underline that this is not an Org resource. Maybe this was the original intention of "VDC"?

Reply:
Improved overall message